packages/react/src/config.ts

```diff
@@ -57,6 +58,9 @@ export type LensConfig = {
    * @see {@link appId} helper
    */
   appId?: AppId;
+
+  /** TODO */
+  mediaTransforms?: MediaTransformsConfig;
```

Comment:
new config where user can define transformations per type of media they want to receive